doc/crypto/overview/functionality.rst

```diff
@@ -134,28 +135,16 @@ Most implementations provide the following functions:
     designed to work only with short-term keys, or only with long-term
     non-extractable keys, do not need to provide this function.
 
-Symmetric cryptography
-~~~~~~~~~~~~~~~~~~~~~~
+Cryptographic operations
+------------------------
 
-This specification defines interfaces for the following types of symmetric
-cryptographic operation:
+For each type of cryptographic operation, the API can include:
 
-*   Message digests, commonly known as hash functions. See :secref:`hashes`.
-*   Message authentication codes (MAC). See :secref:`macs`.
-*   Symmetric ciphers. See :secref:`ciphers`.
-*   Authenticated encryption with associated data (AEAD). See :secref:`aead`.
-*   Key derivation. See :secref:`kdf`.
-
-For each type of symmetric cryptographic operation, the API can include:
-
-*   A pair of *single-part* functions. For example, compute and verify, or
-    encrypt and decrypt.
-*   A series of functions that permit *multi-part operations*.
-
-Key derivation only provides multi-part operation, to support the flexibility required by these type of algorithms.
+*   One or more *single-part* functions, that carry out one function of that operation. For example, compute, verify, encrypt, or decrypt.
```

Comment:
Nit / editorial: I found this wording slightly confusing. Would something like:

```suggestion
*   One or more *single-part* functions, which perform a whole operation in a single function call. For example, compute, verify, encrypt or decrypt.
```
Work as an alternative?